### ISSUE TITLE
Note on allowSyntheticDefaultImports

### ISSUE THREAD

**Issue by henrikra** [ds300/react-native-typescript-transformer#47]
In readme it reads:

> This is neccessary until TypeScript implements suport for synthetic default imports in emitted code as well as in the type checker. See Microsoft/TypeScript#9562.

That issue is now closed so I guess the README should change too?

**Comment by ds300** [comment]
Yes, thanks for the reminder! First I'll update the transformer to be able to run without babel if TS is >= 2.7

**Comment by ds300** [comment]
update on this: Last night I ripped out the babel transform, and ran in to this issue: https://github.com/Microsoft/TypeScript/issues/21621

Once that bug is solved I'll be happy to release react-native-typescript-transformer 2.0 which won't use babel at all and will only support React Native >= 0.52

wip branch over here: https://github.com/ds300/react-native-typescript-transformer/tree/no-babel

**Comment by henrikra** [comment]
Sounds good!

**Comment by henrikra** [comment]
@ds300 Looks like the issue is closed? So we are getting rid of babel now? :O

**Comment by skellock** [comment]
What a roller coaster.

* Babel 7 will support TS! 😮 🎈🎉 
* but not all of it 😭 💥 
* Hey, let's remove babel!  😮 🎈🎉 
* But we can't cuz of a TS bug.  😢 💥 
* It's fixed!:  😮 🎈🎉 
* but we have to wait for 2.9  😪💥 

You're killing me here David.

**Comment by ds300** [comment]
https://twitter.com/typescriptlang/status/1002230349521338368?s=19

🎊💃🕺🎉

Unfortunately I'm busy with non-tech stuff for a few days, but I'll work on a new release ASAP.

**Comment by JReinhold** [comment]
hey @ds300 , how's the 2.0 coming along? Honestly I'm just unsure if I should set `"module": "es2015"` in `tsconfig` or not, based on the info in the README.
